llama/generation.py

```diff
@@ -265,7 +273,7 @@ def chat_completion(
                 f"{B_INST} {(dialog[-1]['content']).strip()} {E_INST}",
                 bos=True,
                 eos=False,
-            )
+            ).to(default_device())
```

Comment:
AttributeError on this line: 'list' object has no attribute 'to'

Reply:
thank you, resolved, 

somehow this was only impacting example_chat_completion.py only so I missed it

however for example_chat_completion.py there are assertions that are failing, does not look like it has anything to do with my changes. I can make it work but this will require:
* removing assert
* explicitly adjust tensor shape casting

neither of these changes should be committed (but I still can if needed)